spec/views/posts/new.html.erb_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe "posts/new", type: :view do
+  before(:each) do
+    assign(:post, Post.new(
+      title: "MyString",
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>[Correctable] Layout/FirstArgumentIndentation: Indent the first argument one step more than Post.new(.